### DISCUSSION TITLE
Close Dropdown Menu on Click

### DISCUSSION THREAD

**Discussion by renbesson** [saadeghi/daisyui#973] [Q&A] [answered]
Hi Everyone!

I've searched everywhere but couldn't find an answer.

The question is simple. How can I close the dropdown once I click its content?

https://daisyui.com/components/dropdown/

Appreciate any suggestions and Thanks!

**Comment by joelelo** [comment]
you can add on click event on the ul tag or the contents that blurs the ul tag (the one with tabindex). [https://www.w3schools.com/jsref/met_html_blur.asp](url)

**Reply by renbesson** [reply]
Thanks! I will try that later.

**Comment by saadeghi** [comment, marked as answer]
See #201

**Reply by renbesson** [reply]
Thank you! I will try that. Sorry I didn't see that discussion before. It's exactly my question. 🤦‍♀️
